MathLib/LinAlg/Eigen/EigenMapTools.h

```diff
@@ -188,6 +188,28 @@ Eigen::Map<Vector> toVector(std::vector<double>& data,
     return {data.data(), size};
 }
 
+/*! Creates an Eigen mapped vector from the given data vector.
```

Comment:
Only the format is changed in this file. Please use ``git checkout  [ufz/master] MathLib/LinAlg/Eigen/EigenMapTools.h``, and then use git add and git commit --amend  to recovery it to its origin.